src/main/scala/scorex/core/network/NodeViewSynchronizer.scala

```diff
@@ -224,23 +234,25 @@ class NodeViewSynchronizer[TX <: Transaction,
       val modifiers = data._2
 
       log.info(s"Got modifiers of type $typeId from remote connected peer: $remote")
-      log.trace(s"Received modifier ids ${data._2.keySet.map(encoder.encode).mkString(",")}")
+      log.trace(s"Received modifier ids ${data._2.keySet.map(id => encoder.encode(id)).mkString(",")}")
```

Comment:
Do we need `encoder.encode` here?

Reply:
I think it might be useful if we'll decide to switch from Base16 in ModifierId to something else, or if desired enocoding will be different from Base16